pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -478,14 +479,14 @@ def __resolve_ckpt_dir(self, trainer, pl_module):
             version, name = trainer.accelerator_backend.broadcast((version, trainer.logger.name))
 
             ckpt_path = os.path.join(
-                save_dir, name, version, "checkpoints"
+                save_dir, str(name), version, "checkpoints"
```

Comment:
why is `str` required?

Reply:
in DummyLogger version is None, so this worked out :smile: